content/en/news/releases/1.19.x/announcing-1.19/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.19.0
+linktitle: 1.19.0
+subtitle: Major Release
+description: Istio 1.19 Release Announcement.
+publishdate: 2023-09-01
+release: 1.19.0
+aliases:
+    - /news/announcing-1.19
+    - /news/announcing-1.19.0
+---
+
+We are pleased to announce the release of Istio 1.19. This is the third Istio release of 2023. We would like to thank the entire Istio community for helping get the 1.19.0 release published. We would like to thank the Release Managers for this release, `Kalya Subramanian` from Microsoft, `Xiaopeng Han` from DaoCloud, and `Aryan Gupta` from Google. The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.19.0 is officially supported on Kubernetes versions `1.25` to `1.28`.
+{{< /tip >}}
+
+## What's new
+
+TODO(ksubrmnn)
+
+## Upgrading to 1.19
```

Comment:
We should add an upgrade note for https://github.com/kubernetes-sigs/gateway-api/issues/2309